projects/ti/cc3220_launchpad/iar/aws_tests/aws_tests.eww

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+
+<workspace>
+  <project>
+    <path>$WS_DIR$\aws_tests.ewp</path>
+  </project>
+  <batchBuild>
+    <batchDefinition>
+      <name>all</name>
+      <member>
+        <project>aws_tests</project>
+        <configuration>Debug</configuration>
+      </member>
+      <member>
+        <project>aws_tests</project>
+        <configuration>Release</configuration>
```

Comment:
Do we have Release configuration as well?

Reply:
I don't know.  This was generated through IDE generator